core/src/main/java/org/apache/iceberg/util/DateTimeUtil.java

```diff
@@ -88,4 +88,38 @@ public static long microsFromTimestamptz(OffsetDateTime dateTime) {
   public static String formatTimestampMillis(long millis) {
     return DATE_FORMAT.format(LocalDateTime.ofInstant(Instant.ofEpochMilli(millis), ZoneOffset.UTC));
   }
+
+  public static String formatEpochDays(int days) {
+    return LocalDate.ofEpochDay(days).format(DateTimeFormatter.ISO_LOCAL_DATE);
+  }
+
+  public static String formatTimeOfDayMicros(long micros) {
```

Comment:
Could you name these `microsToIsoTime` and `isoTimeToMicros`? I think the shorter names are more clear.

Reply:
done